### PR TITLE
Added view that shows info about all teams

### LINKED ISSUE #26: General ranking list
Also has a filter option for calculating according to specific game specs.

Possibly, make a sort of alternate scoring system to decide which team is better.

### PR DESCRIPTION
**Pull request by TheArchNemesis**
closes #26

### CODE REVIEW

File: src/components/charts/detailed-bar.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }}
+    width='100%'
+    height='30vh'
+
+    options={{
+      maintainAspectRatio: true,
+      scales: {
+        yAxes: [{
+          ticks: {
+            min: 0
```
Comment (YonatanNitzan) [inline src/components/charts/detailed-bar.js]: Add stepSize = 1

File: src/views/general-info.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  render () {
+    if (this.state.data) {
+      return (
+        <div className="text-center" style={{
+          margin: '20px'
+        }}>
+          <MainMenu view="general-info"/><br/>
+          <GraphSet config="" parser={a => {
```
Comment (YonatanNitzan) [inline src/views/general-info.js]: Add parser and config, this is highly unorganized.

File: src/components/charts/detailed-bar.js (hunk 1 above)
Comment (TheArchNemesis) [reply src/components/charts/detailed-bar.js]: That looks absolutely ugly when a chart reaches a value of like 30...

File: src/views/general-info.js (hunk 2 above)
Comment (TheArchNemesis) [reply src/views/general-info.js]: I agree, no need for config, just a "parser". most of the parsing happens on the server side...

File: src/components/charts/detailed-bar.js (hunk 1 above)
Comment (YonatanNitzan) [reply src/components/charts/detailed-bar.js]: StepSize sets the minimum tho, not the max stepSize

Comment (YonatanNitzan) [reply src/components/charts/detailed-bar.js]: See http://www.chartjs.org/docs/latest/axes/cartesian/linear.html#step-size

File: src/views/general-info.js (hunk 2 above)
Comment (YonatanNitzan) [reply src/views/general-info.js]: So why do you need a parser and not a config?

File: src/components/charts/detailed-bar.js (hunk 1 above)
Comment (TheArchNemesis) [reply src/components/charts/detailed-bar.js]: That is the documentation for suggestedMin and Max tho, the step size is constant...

Comment (YonatanNitzan) [reply src/components/charts/detailed-bar.js]: Ok you're right

**Pull request merged**